packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -266,43 +290,37 @@ abstract contract MajorityVotingBase is
             !vote_.executed;
     }
 
-    /// @notice Calculates whether `_value` is more than a percentage `_pct` of `_total`.
-    /// @param _value the current value.
-    /// @param _total the total value.
-    /// @param _pct the required support percentage.
-    /// @return returns if the _value is _pct or more percentage of _total.
-    function _isValuePct(
-        uint256 _value,
-        uint256 _total,
-        uint256 _pct
-    ) internal pure returns (bool) {
+    /// @notice Calculates the relative of value with respect to a total as a percentage.
+    /// @param _value The value.
+    /// @param _total The total.
+    /// @return returns The relative value as a percentage.
+    function _calculatePct(uint256 _value, uint256 _total) internal pure returns (uint256) {
         if (_total == 0) {
-            return false;
+            revert ZeroValueNotAllowed();
         }
 
-        uint256 computedPct = (_value * PCT_BASE) / _total;
-        return computedPct > _pct;
+        return (_value * PCT_BASE) / _total;
     }
 
     function _validateAndSetSettings(
-        uint64 _participationRequiredPct,
-        uint64 _supportRequiredPct,
+        uint64 _totalSupportThresholdPct,
+        uint64 _relativeSupportThresholdPct,
         uint64 _minDuration
     ) internal virtual {
-        if (_supportRequiredPct > PCT_BASE) {
-            revert VoteSupportExceeded({limit: PCT_BASE, actual: _supportRequiredPct});
+        if (_relativeSupportThresholdPct > PCT_BASE) {
+            revert PercentageExceeds100({limit: PCT_BASE, actual: _relativeSupportThresholdPct});
         }
 
-        if (_participationRequiredPct > PCT_BASE) {
-            revert VoteParticipationExceeded({limit: PCT_BASE, actual: _participationRequiredPct});
+        if (_totalSupportThresholdPct > PCT_BASE) {
+            revert PercentageExceeds100({limit: PCT_BASE, actual: _totalSupportThresholdPct});
         }
 
         if (_minDuration == 0) {
             revert VoteDurationZero();
         }
 
-        participationRequiredPct = _participationRequiredPct;
-        supportRequiredPct = _supportRequiredPct;
+        totalSupportThresholdPct = _totalSupportThresholdPct;
+        relativeSupportThresholdPct = _relativeSupportThresholdPct;
```

Comment:
These magnitudes should not be percentages (0-100), they should be **ratios** of a predefined size (0 .. RATIO_MAX)

Reply:
This will be worked on in task https://aragonassociation.atlassian.net/browse/APP-1264